build.gradle

```diff
@@ -158,21 +135,25 @@ dependencyCheck {
 
 ext {
   log4JVersion = "2.17.0"
+  feignJackson = '11.6'
+  groovyVersion = '3.0.11!!'
+  hamcrestVersion = '1.3'
+  hibernateValidatorVersion = '6.0.20.Final'
+  jacksonVersion = '2.13.2'
+  javaxValidationVersion = '2.0.1.Final'
+  junitJupiterVersion = '5.8.1'
+  junitVintageVersion = '5.7.0'
+  log4jVersion = '2.17.1'
+  postgresqlVersion = '42.4.1'
+  powermockVersion = '2.0.0-beta.5'
+  reformLogging= '5.1.9'
   restAssuredVersion = '4.3.0!!'
-  lombokVersion = "1.18.24"
-  junit_version = "4.12"
-  junitJupiterVersion = '5.7.1'
-  junitVintageVersion = '5.7.1'
-  powermockVersion = '2.0.7'
-  springSecurity   =  '5.7.4'
+  snakeyamlVersion = '1.32'
```

Comment:
these are needed as this will override the transitive dependencies versions from gradle parent modules
and will address CVE issue related to snakeyaml version

Reply:
removed snakeyamlVersion

Reply:
Confirmed that variable has been removed.